go/runtime/txpool/txpool.go

```diff
@@ -460,20 +466,49 @@ func (t *txPool) checkTxBatch(ctx context.Context, rr host.RichRuntime) {
 		return
 	}
 
+	// Check if node is synced.
+	var notReady bool
+	select {
+	case <-t.consensus.Synced():
+		// Node is synced.
+	default:
+		// Node is not synced.
+		notReady = true
+		t.logger.Debug("consensus is not synced yet")
+	}
+
+	if !notReady {
+		// Node is synced, ensure block round is synced to storage.
+		waitSyncCtx, cancelWaitSyncCtx := context.WithTimeout(ctx, checkTxWaitRoundSyncedTimeout)
+		defer cancelWaitSyncCtx()
+
+		t.logger.Debug("ensuring block round is synced", "round", bi.RuntimeBlock.Header.Round)
```

Comment:
```suggestion
		t.logger.Debug("ensuring runtime block is synced", "round", bi.RuntimeBlock.Header.Round)
```

Reply:
I would prefer to retain the original debug message, since I didn't write it.